wiki/Tournaments/Official_support/en.md

```diff
@@ -40,6 +40,8 @@ Community-run tournaments which abide by the following hard criteria are eligibl
   - osu!mania: 40,000
   - This does not apply to open rank tournaments.
 - The tournament must adhere to all [expectations](#expectations) of officially supported tournaments.
+  - Any tournaments held as preliminary events leading into a main competition must adhere to the same [expectations](#expectations) as standalone tournaments.
+  - The forum threads for the preliminary events MUST contain a clearly visible link in a normal-sized font to the [tournament reports form](https://pif.ephemeral.ink/tournament-reports) as the very last content of the original post.
```

Comment:
does this point really need to be duplicated?

Reply:
While it may seem superfluous, not all eligibility criteria are required for preliminary events in cases where the preliminary events are not seeking to be officially supported. Consider [The Roundtable II Qualifier Tournament](<https://osu.ppy.sh/community/forums/topics/1699086>) for example.  
However, even if the other eligibility requirements aren't strictly necessary, we do still necessitate a link to the report form to incentivise reporting any breaches of expectations.

Perhaps if we were to merge it into the already existing line, that would be preferred? Something akin to "The forum threads associated with the tournament — this includes forum threads for any preliminary events — MUST ...".

Reply:
> While it may seem superfluous, not all eligibility criteria are required for preliminary events in cases where the preliminary events are not seeking to be officially supported. Consider [The Roundtable II Qualifier Tournament](https://osu.ppy.sh/community/forums/topics/1699086) for example.

then what eligibility criteria apply for preliminary events? none of this is mentioned anywhere else in the article

> Perhaps if we were to merge it into the already existing line, that would be preferred? Something akin to "The forum threads associated with the tournament — this includes forum threads for any preliminary events — MUST ...".

yeah i agree